src/physics/src/spalart_allmaras_helper.C

```diff
@@ -0,0 +1,152 @@
+//-----------------------------------------------------------------------bl-
+//--------------------------------------------------------------------------
+//
+// GRINS - General Reacting Incompressible Navier-Stokes
+//
+// Copyright (C) 2014-2015 Paul T. Bauman, Roy H. Stogner
+// Copyright (C) 2010-2013 The PECOS Development Team
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the Version 2.1 GNU Lesser General
+// Public License as published by the Free Software Foundation.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc. 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301  USA
+//
+//-----------------------------------------------------------------------el-
+
+
+// This class
+#include "grins/spalart_allmaras_helper.h"
+
+// GRINS
+#include "grins/assembly_context.h"
+#include "grins/generic_ic_handler.h"
+#include "grins/inc_navier_stokes_bc_handling.h"
+#include "grins/turbulence_models_macro.h"
+
+// libMesh
+#include "libmesh/quadrature.h"
+#include "libmesh/elem.h"
+#include "libmesh/unstructured_mesh.h"
+#include "libmesh/fem_system.h"
+
+namespace GRINS
+{
+  
+  SpalartAllmarasHelper::SpalartAllmarasHelper(const GetPot& input )
+    : _cb1(0.1355), // Define class variables
+      _sigma(2./3.),
+      _cb2(0.622),
+      _kappa(0.41),
+      _cv1(7.1),
+      _cv2(0.7),
+      _cv3(0.9),
+      _r_lin(10.0),
+      _c_w2(0.3),
+      _c_w3(2.0),
+      _c_t3(1.2),
+      _c_t4(0.5),
+      _c_n1(16.0),
```

Comment:
@trevilo @vikramvgarg can you comment on whether we want to to be able to turn those knobs, either at input or in a sensitivity study?